shop/views/checkout.py

```diff
@@ -248,7 +248,6 @@ def get(self, request, *args, **kwargs):
         return super(OrderConfirmView, self).get(request, *args, **kwargs)
 
     def get_redirect_url(self, **kwargs):
-        self.url = reverse(self.url_name)
```

Comment:
[Django `RedirectView` does it for you in `get_redirect_url` if you set `pattern_name` instead of `url`](https://github.com/django/django/blob/master/django/views/generic/base.py#L178).

The `OrderConfirmView` duplicates `RedirectView.get_redirect_url`'s behaviour for when `pattern_name` is set, but incorrectly applies this behaviour to `url`.